packit_service/worker/handlers/copr.py

```diff
@@ -415,6 +416,18 @@ def handle_testing_farm(self):
                 if (
                     not job_config.skip_build
                     and not job_config.manual_trigger
+                    and (
+                        job_config.trigger != JobConfigTriggerType.pull_request
```

Comment:
Probably too late to the party, but `job_config.require.label.satisfied(job_config, self.copr_build_helper.pull_request_obejct` would've been nice, I have a feeling that something like this is in the other PR too.

Maybe `NOTE`-worthy? :D